From 11dc017e902397c452331425eb6101b3315572fa Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sun, 24 Jan 2021 10:59:31 +0100
Subject: [PATCH] Makefile.PL: fix cross-compilation with gdlib.pc

Cross-compilation will fail if gdlib.pc does not contain any cflags.
Indeed, if cflags is empty, Makefile.PL will use the default value for
INC (i.e. -I/usr/include -I/usr/include/gd)

It should be noted that gdlib-config has been dropped from gd since
version 2.3.0

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: probably not upstreamable]
---
 Makefile.PL | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index 25f2f93..7da3651 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -47,7 +47,7 @@ If you want to try to compile anyway, please rerun this script with the option -
 END
 }
 
-@INC     = qw(-I/usr/include -I/usr/include/gd) unless @INC;
+#@INC     = qw(-I/usr/include -I/usr/include/gd) unless @INC;
 @LIBPATH = qw(-L/usr/lib/X11 -L/usr/X11R6/lib -L/usr/X11/lib -L/usr/lib) unless @LIBPATH;
 @LIBS    = qw(-lgd) unless @LIBS;
 
-- 
2.29.2

